### ISSUE TITLE
Latin n (characters) result to an error

### ISSUE THREAD

**Issue by pvperez1** [CollectionBuilder/collectionbuilder-csv#28]
My metadata contains a latin n, when viewed in vscode, it appears as a question mark. Also, when bundle exec jekyll s was run, the site do not generate and results to an error. How can we present latin or accented characters? Thanks!
<img width="56" alt="Screenshot 2022-04-07 012509" src="https://user-images.githubusercontent.com/29356154/162032794-214400f2-f6f7-4f0e-bc29-ff7b5fae659f.png">

**Comment by evanwill** [comment]
@pvperez1 sounds like a file character encoding issue. The question mark character is displayed when a character isn't encoded correctly (or is being interpreted in the wrong encoding), so the editor doesn't know what character to display. The corrupted characters shouldn't break anything, however, a CSV created by Excel often can (the encoding “UTF-8 with BOM” will cause errors in Ruby). 

Can you point me to the repository with the issue?

Basic notes:

The metadata CSV needs to be in UTF-8 (*without BOM*). There are some tips on: https://collectionbuilder.github.io/cb-docs/docs/metadata/uploading/
UTF-8 can encode any characters--so if formatted correctly, you can display any kind of accents, etc. 

Sometimes you can fix the CSV by re-encoding on VS Code and choosing UTF-8 (click the encoding such as "UTF-8" at the bottom of the editor window, displayed in the blue bar at the bottom right). 
However, sometimes the characters are fully corrupted in the CSV (because it is interpreting it in the wrong encoding to start with)--so you might need to go back to the source spreadsheet and export it again, or figure out what the encoding really is.
If you create metadata in Excel--don't try to save the CSV from Excel, it doesn't work! Import the xlsx file into Sheets or OpenRefine or LibreOffice, then create the CSV from one of those tools instead.

**Comment by evanwill** [comment]
Yeah, @pvperez1 Excel won't properly parse the UTF-8 csv (if you have any special characters in it, they will be corrupted) *and* can't properly export a csv--you can create metadata directly in Excel, using the xlsx file format then export using something else--but going back and forth with CSV + Excel doesn't work. 
If you want a desktop app for editing spreadsheets, I would use LibreOffice (its free and open). There is also some plugins for VS Code for editing spreadsheets--but I am not familiar with them, so can't suggest anything! I do a lot of csv editing just directly in Code, not so hard once you get used to looking at it. I use the [Rainbow CSV](https://marketplace.visualstudio.com/items?itemName=mechatroner.rainbow-csv) extension to make visualizing the columns easier.
